crates/omnix-cli/src/command/mod.rs

```diff
@@ -23,6 +35,11 @@ impl Command {
             Command::Init(config) => config.run().await,
             Command::CI(config) => config.run().await,
             Command::Health(config) => config.run().await,
+            Command::Completion { shell } => {
+                let mut cli = Args::command();
+                generate(*shell, &mut cli, "om", &mut io::stdout());
+                Ok(())
+            }
```

Comment:
Move all of this to `command/completion.rs` to be consistent with the rest of the subcommands.